homeassistant/components/sensor/melissa.py

```diff
@@ -0,0 +1,92 @@
+"""
+Support for Melissa climate Sensors.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.melissa/
+"""
+import logging
+
+from homeassistant.components.melissa import DOMAIN, DATA_MELISSA
+from homeassistant.const import TEMP_CELSIUS, STATE_UNKNOWN
+from homeassistant.helpers.entity import Entity
+
+DEPENDENCIES = [DOMAIN]
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Setup the melissa sensor platform."""
+    sensors = []
+    api = hass.data[DATA_MELISSA]
+    devices = api.fetch_devices().values()
+    append = sensors.append
```

Comment:
I would just use `sensors.append` directly in the loop, no need for introducing a variable for that (and it will make it easier for grep etc.).

Reply:
The reason I did it this way is https://wiki.python.org/moin/PythonSpeed/PerformanceTips

But I do agree that its more of a language problem that it should be avoided, Ill change it to use .append in the loop